cmd/sdkstats/main.go

```diff
@@ -7,46 +7,188 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
+	"regexp"
+	"strings"
 
-	"github.com/go-github-metrics-1/pkg/sdkstats"
+	"github.com/learnoperators/go-github-metrics/pkg/sdkstats"
 
 	"github.com/google/go-github/github"
 	"golang.org/x/oauth2"
 )
 
+type helmVersionParser struct{}
```

Comment:
As a good practice, we should have basic info in the README in order to allow any person knows how to configure use the project. It could be another PR, however, how we can review this one without the knowledge to test it locally?